packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -45,6 +43,8 @@ @interface FLTImagePickerPlugin () <UINavigationControllerDelegate,
 typedef NS_ENUM(NSInteger, ImagePickerClassType) { UIImagePickerClassType, PHPickerClassType };
 
 @implementation FLTImagePickerPlugin {
+  NSDictionary *_arguments;
```

Comment:
I believe @jmagman prefers we move *to* using properties for everything, rather than away from properties in favor of raw ivars.

Reply:
Moved back to using properties, my bad (Jenn indeed explained this to me once before).